docker-compose.test.yml

```diff
@@ -0,0 +1,33 @@
+version: "2.1"
+services:
+  postgres:
+    image: postgres:14-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: docker
+      POSTGRES_PASSWORD: docker
+      POSTGRES_DB: postgres
+    ports:
+      - "5432"
+      - "5431:5432"
```

Comment:
Why do we expose postgres on both 5432 and 5431?

Reply:
Exposing only 5431, thx!